frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/Home/LanguageSelection.js

```
@@ -1,48 +1,399 @@
-    [selectedLanguage, languages]
-  );
+      <div style={styles.banner}>
+        <img
+          src="https://tfstate8auyj.blob.core.windows.net/egov-dev-assets/Rectangle%2010.svg"
+          alt="Rajwada"
+          style={styles.image}
+        />
+        <div style={styles.bannerText}>
+          <h2 style={styles.yourUrban}>Your Urban Services<br />One Login Away!</h2>
+          <div style={styles.loginButtons}>
+            <button style={styles.loginButton} > <img src="https://tfstate8auyj.blob.core.windows.net/egov-dev-assets/Vector.svg" />{" "} Citizen Login</button>
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)